core/src/main/java/com/oath/oak/OakRBuffer.java

```diff
@@ -135,4 +135,12 @@
      * @return The return value of the transform
      */
     <T> T transform(Function<ByteBuffer, T> transformer);
+
+    /**
+     * Perform an unsafe memcopy from the underlying buffer to the dst int array.
+     * @param srcPosition The position in the underlying buffer.
+     * @param dstArray The int array to copy to.
+     * @param countInts The number of ints to copy.
+     */
+    void unsafeBufferToIntArrayCopy(int srcPosition, int[] dstArray, int countInts);
```

Comment:
So this is a new api method to be used by the application? and you are using it in Druid to improve performance?
If so, it would be helpful to add a few sentences on when it is advisable to use this method. Either here or in the class documentation, or in both places.

Reply:
Currently used in druid. hasn't shown any major performance impact on the benchmarks we are running